packages/core/src/browser/navigatable.ts

```diff
@@ -82,7 +83,8 @@ export abstract class NavigatableWidgetOpenHandler<W extends NavigatableWidget>
     protected createWidgetOptions(uri: URI, options?: WidgetOpenerOptions): NavigatableWidgetOptions {
         return {
             kind: 'navigatable',
-            uri: this.serializeUri(uri)
+            uri: this.serializeUri(uri),
+            contents: uri.contents
```

Comment:
why contents is a part of options which unique identiry it?

Reply:
Please see https://github.com/eclipse-theia/theia/pull/8589#discussion_r524977010

Reply:
@akosyakov @vince-fugnitto 
cc: @amiramw 
Reverted this changes - removed "contents" from uri.